$ python3 main.py --dataset_directory ../data --dataset spam ann --layer_sizes 0 --max_iters 5000
{'classifier': 'ann', 'schema': <data.Schema object at 0x7f7e38310e80>, 'max_iters': 5000, 'flip_probability': 0, 'epsilon': None, 'gamma': 0.01, 'layer_sizes': 0, 'num_hidden': 40}
Traceback (most recent call last):
  File "main.py", line 229, in <module>
    main(**vars(args))
  File "main.py", line 159, in main
    classifier = get_classifier(**options)
  File "main.py", line 57, in get_classifier
    return CLASSIFIERS[classifier_name](**options)
TypeError: __init__() got an unexpected keyword argument 'flip_probability'
Ran for 0:00:01.284483 (H:MM:SS.fraction)














